apps/prairielearn/src/lib/workspace.js

```diff
@@ -451,6 +486,19 @@ module.exports = {
       }
     }
 
+    issues.forEach(async (issue) => {
```

Comment:
Rather than creating one issue per "issue", I'd consider constructing a single issue with a multi-line description, e.g.:

```
foo/bar.txt: Dynamic workspace file points to a local file outside the question directory. File ignored.
bar/baz.txt: Workspace file could not be written to workspace.
```

I doubt either students or instructors would benefit from having 10 issues created if for some reason we failed to copy 10 files.

Reply:
Done in https://github.com/PrairieLearn/PrairieLearn/commit/a83dae3c517de632b0c064eb5dfa762ce80c94bd.